composer.json

```diff
@@ -14,23 +14,24 @@
     "require": {
         "php": ">=7.2",
         "doctrine/doctrine-bundle": "^1.8|^2.0",
-        "doctrine/orm": "^2.6",
+        "doctrine/orm": "^2.7|^3.0",
```

Comment:
The constraint should be just `^2.7`. We can't claim to support 3.0 as it hasn't even been released yet and who knows what breaking changes will yet be made there.

Reply:
fixed, thank you!